cypress/integration/homework_spec.ts

```diff
@@ -0,0 +1,33 @@
+///<reference types="Cypress" />
+
+import {HomePO} from "../support/Home.PO";
+import {SearcDetailsPO} from "../support/SearcDetails.PO";
+import {HomeWorkPO} from "../support/HomeWork.PO";
+
+describe('HOME WORK Gift card test', () => {
+    const HW = new HomeWorkPO();
+    beforeEach(() => cy.visit(HW.pageUrl));
+
+    it('gift card test', () => {
+        const giftCardsUrlPart = '/gift-cards/';
+        const ProductUrlPart = 'product';
+        const price = '100'
```

Comment:
in this case important notice, that price in $, so need to check that this symbol is shown also